ui/src/serverIO.js

```diff
@@ -105,7 +107,7 @@ class ServerIO {
     this.hwrSocket.on('disconnect', (reason) => {
       console.log('hwrSocket disconnected!'); // eslint-disable-line no-console
 
-      setTimeout(() => {
+      this.connectionLostTimeout = setTimeout(() => {
```

Comment:
This timeout was triggered sometimes when logging out and back in quickly, which made the connection-lost dialog flash on the screen.

Reply:
Well spotted, nice fix !